dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataset/DatasetReportService.java

```diff
@@ -109,11 +108,19 @@ private Record createDatasetRecord(Dataset dataset) {
     private void processDatasetVersion(DatasetVersion datasetVersion, CSVPrinter csvPrinter, Record datasetRecord) throws IOException {
         List<FileMetadata> allFilesMetadataSorted = datasetVersion.getAllFilesMetadataSorted();
         Record datasetVersionRecord = createDatasetVersionRecord(datasetVersion, datasetRecord);
+        datasetVersionRecord.setDatasetTitle(getDatasetTitleInVersion(datasetVersion));
+
         for (FileMetadata fileMetadata : allFilesMetadataSorted) {
+            datasetVersionRecord.setFileName(fileMetadata.getLabel());
             processFile(fileMetadata, csvPrinter, datasetVersionRecord);
         }
     }
 
+    private String getDatasetTitleInVersion(DatasetVersion datasetVersion) {
+        return datasetVersion.getDatasetFieldByTypeName(DatasetFieldConstant.title).isPresent() ?
+            datasetVersion.getDatasetFieldByTypeName(DatasetFieldConstant.title).get().getValue() : datasetVersion.getDataset().getDisplayName();
```

Comment:
Hmm aren't those two methods almost the same? That's the code for displayname -> getTitle 
```
or (DatasetField dsfv : this.getDatasetFields()) {
            if (dsfv.getDatasetFieldType().getName().equals(DatasetFieldConstant.title)) {
                retVal = dsfv.getDisplayValue();
            }
        }
```
I think I would just use datasetVersion.getDataset().getDisplayName() for simplicity.

Reply:
I'd rather avoid using `displayValue` methods in report if I can help it. They depend on field display format which is part of metadata. It's usually not a problem when a given field has format like `#VALUE` but if it changes the problem occurs - as I have lately raised this issue when we have changed display format for external Ids identifiers and exporters started to print data incorrectly..

Here I only fallback for `displayValue` and I can switch `datasetVersion.getDataset().getDisplayName();` to `datasetVersion.getTitle()`

Reply:
Ahh ok then.